pages/mobile/search.py

```diff
@@ -12,19 +12,23 @@
 
 class Search(Base):
 
-    _page_url = '/en-US/search'
     _results_locator = (By.CSS_SELECTOR, 'ol.search-results li')
 
-    def __init__(self, testsetup, search_term):
+    def __init__(self, testsetup):
         Base.__init__(self, testsetup)
-        self._page_title = "%s :: Search :: Add-ons for Firefox" % search_term
+        self._page_title = 'Search | Mozilla Support'
 
     @property
     def results(self):
-        return [self.SearchResult(self.testsetup, web_element)
-                for web_element in self.selenium.find_elements(*self._results_locator)]
+        return [self.SearchResult(self.testsetup, element)
+                for element in self.selenium.find_elements(*self._results_locator)]
```

Comment:
We use `element` here but `web_element` in `Base.menu_items`, let's be consistent (I prefer `element`)